src/test/java/org/fundacionjala/pivotal/cucumber/steps/StoriesSteps.java

```diff
@@ -1,66 +1,42 @@
 package org.fundacionjala.pivotal.cucumber.steps;
 
 import cucumber.api.java.en.And;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'cucumber.api.java.en.And'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)